spoc/io.py

```diff
@@ -73,12 +104,149 @@ def load_chromosome_sizes(path: str):
         )
 
     @staticmethod
-    def load_pixels(path: str):
-        """Loads pixels"""
-        # TODO:make this conform with new file format
-        raise NotImplementedError
+    def _load_metadata(path: str):
+        """Load metadata"""
+        metadata_path = Path(path) / "metadata.json"
+        if metadata_path.exists():
+            with open(metadata_path, "r", encoding="UTF-8") as f:
+                metadata = json.load(f)
+        else:
+            raise ValueError(f"Metadata file not found at {metadata_path}")
+        return metadata
+
+    @staticmethod
+    def list_pixels(path: str) -> List[PixelParameters]:
+        """List available pixels
+
+        Args:
+            path (str): Path to the pixel data.
+
+        Returns:
+            List[PixelParameters]: List of PixelParameters objects.
+        """
+        # read metadata.json
+        metadata = FileManager._load_metadata(path)
+        # instantiate pixel parameters
+        pixels = [PixelParameters(**params) for params in metadata.values()]
+        return pixels
+
+    @staticmethod
+    def list_contacts(path: str) -> List[ContactsParameters]:
+        """List available contacts"""
+        # read metadata.json
+        metadata = FileManager._load_metadata(path)
+        # instantiate pixel parameters
+        contacts = [ContactsParameters(**params) for params in metadata.values()]
+        return contacts
+
+    def load_pixels(
+        self, path: str, global_parameters: PixelParameters, load_dataframe: bool = True
```

Comment:
Same applies to `load_contacts`: Why use a separate class for parameters instead of writing all kwargs? I think it is quite rare and not quite pythonic to do that. And also might be confusing for users, especially if this is the only place that requires specifying global_parameters.

Reply:
Move the from URI method here.